src/peer_connection_configuration/configuration.c

```diff
@@ -258,3 +258,18 @@ enum rawrtc_code rawrtc_peer_connection_configuration_set_sctp_mtu_discovery(
     configuration->sctp.mtu_discovery = on;
     return RAWRTC_CODE_SUCCESS;
 }
+
+enum rawrtc_code rawrtc_peer_connection_configuration_set_ice_udp_port_range(
+    struct rawrtc_peer_connection_configuration* configuration,
+    uint16_t min_port,
+    uint16_t max_port) {
+    if (!configuration) {
+        return RAWRTC_CODE_INVALID_ARGUMENT;
+    }
+    if (max_port <= min_port) {
```

Comment:
One port is enough I guess :)

```suggestion
    if (max_port < min_port) {
```

Reply:
I thought I tested this and it looked like it was being unhappy if I passed in `max_port == min_port`, but perhaps I wasn't checking things carefully enough. It did seem odd that `trice_set_port_range` would permit `max_port == min_port` in that case. I can double check that though, since I didn't debug it carefully.

Reply:
Pretty sure this was an artifact of using multiple WebRTC channels, so of course it attempted to open extra ports. The errors could probably be clearer, but yeah, `<` is appropriate.